spine/model/update.go

```diff
@@ -0,0 +1,64 @@
+package model
+
+import "github.com/DerAndereAndi/eebus-go/util"
+
+type Updater interface {
+	DoUpdate()
+}
+
+type UpdaterFactory[T any] interface {
+	NewUpdater(s *T, filterPartial *FilterType, filterDelete *FilterType) Updater
+}
+
+type UpdateDataProvider[T util.HashKeyer] interface {
+	ExistingData() []T
+	NewData() []T
+	UpdateSelektorHashKey() *string
```

Comment:
Shouldn't the method be named `UpdateSelectorHashKey` ?

Reply:
Fixed